Day_2/04_DifferentialExpression.Rmd

```diff
@@ -120,7 +119,7 @@ dgeList <- countData %>%
 
 Here we have taken our counts table with gene IDs as row names and sample IDs as column names and put it in to a `DGEList` object, called `dgeList`, and called the `calcNormFactors()` function to calculated the normalisation factors, which can be seen in `dgeList$samples`.
 
-If we decided to fit our counts using a negative binomial model (for discrete data), these `norm.factors` would be included in the model to adjust for variations in the library size and count distributions (for an interesting post on what "fitting" a model is see [fit] (https://diamondage.com/2017/06/03/what-does-it-mean-to-fit-a-model-anyway/)).
+If we decided to fit our counts using a negative binomial model (for discrete data), these `norm.factors` would be included in the model to adjust for variations in the library size and count distributions (for an interesting post on what "fitting" a model is see [fit] (https://diamondage.com/2017/06/03/what-does-it-mean-to-fit-a-model-anyway/).
```

Comment:
Remove space between `]` and `(` in linkification and add back terminating non-linkinfication paren.